gcp/src/main/java/org/apache/iceberg/gcp/GCPProperties.java

```diff
@@ -81,6 +87,11 @@ public class GCPProperties implements Serializable {
   private String gcsOauth2RefreshCredentialsEndpoint;
   private boolean gcsOauth2RefreshCredentialsEnabled;
 
+  private String gcsImpersonateServiceAccount;
+  private int gcsImpersonateLifetimeSeconds = GCS_IMPERSONATE_LIFETIME_SECONDS_DEFAULT;
```

Comment:
```suggestion
  private int gcsImpersonateLifetimeSeconds;
```

nit: this is set below